portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,83 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+/* This code script executes when the website is first loaded in  */
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+/* JavaScript "clicks" on the default tab so that content is shown when first going into the website */
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+
+/*global variable that gives me which slide I'm currently on in the SlideShow*/
+var slideIndex;
+
+/* Performs necessary functions when page is first loaded */
+window.onload = function( ){
+    displayDefaultSection();
+    slideIndex = 1;
+    showSlides(slideIndex);
+}
+
+/* Sets the default-tab by "clicking" on the "Background" tab, which has id="default-tab*/
+function displayDefaultSection(){
+    document.getElementById("default-tab").click();
```

Comment:
Let's not simulate a click here. There may be a case where clicking does more than load the tab content and most of the logic in displaySection isn't needed onload and AFAICT you don't actually need the event object since you already know the classname.